src/makerspace/static/makerspace/js/statistics_charts.js

```diff
@@ -0,0 +1,297 @@
+
+Chart.defaults.font.family = "Ubuntu";
+Chart.defaults.font.weight = 500;
+Chart.defaults.font.size = 12.5;
+
+function span_printer_reservations() {
+    const ct1 = $('#span_printer_reservations').get(0).getContext("2d");
+    const data = JSON.parse($('#span_of_printer_reservations').get(0).textContent);
+
+    new Chart(ct1, {
+            type: 'bar',
+            data: {
+                labels: data.map((element)=> element.name),
+                datasets: [{
+                    label: "How long each printer has been reserved in total",
+                    data: data.map((element)=> (element.len)),
+                    backgroundColor: [
+                        'rgba(249, 65, 68)',
```

Comment:
-	I like the colors of the plots. Still, it might be more logical if they were MAKE's official colors, which are MAKE yellow (#F8C811), dark blue (#0C426A) and turquoise (#00EBB3), in addition to two different shades of gray - #4A4B4C and #DCDDDE. We also have an orange variation of the yellow (#F5A13F). I've created some examples. The colors of the gradient were generated using this tool (https://coolors.co/gradient-palette/eeddf3-ee92b1?number=7)
If you decide to keep the colors you already have, maybe the rainbow plot could be smoother, because in the second and fourth bar looks more orange, while the third and the fifth looks more yellow. Also – the ninth bar looks more purple and the tenth looks more blue. On the other hand - the other plots seem to follow a gradient.
Also – if you have 10 colors in your gradient, and we have more than 10 3D printers, we would go from the 10th color and back to the first, which will not look too nice. Maybe the gradient could be calculated in the JavaScript file, from the first and the last color, and the number of bars? If not, it would also be possible to reverse the gradient, as I've shown in the example.

```suggestion
                        '#00EBB3',
                        '#03C1A1',
                        '#06978F',
                        '#096C7C',
                        '#0C426A',
                        '#096C7C',
                        '#06978F',
                        '#03C1A1',
```